build/container/external/packages/aio/00-makefile.patch

```diff
@@ -0,0 +1,43 @@
+--- src/Makefile	2016-11-30 09:59:12.889526296 -0800
```

Comment:
if these patches go out of date (stale), will the application of them during build time fail and subsequently fail the build?